backend/src/main/java/com/htecgroup/skynest/service/impl/FolderServiceImpl.java

```diff
@@ -100,6 +101,23 @@ public FolderResponse getFolderDetails(UUID uuid) {
     return folderResponse;
   }
 
+  @Override
+  public FolderResponse editFolder(FolderEditRequest folderEditRequest, UUID folderId) {
+    FolderEntity folderEntity =
+        folderRepository.findById(folderId).orElseThrow(FolderNotFoundException::new);
+
+    if (folderEntity.getDeletedOn() != null) {
```

Comment:
Would be nice if you make a method for this. I like isDeleted. I think we did it that way before

Reply:
see now if you like it better